platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/AttributionDialogManager.java

```diff
@@ -192,7 +234,7 @@ private void showWebPage(@NonNull String url) {
         for (Source source : mapboxMap.getStyle().getSources()) {
```

Comment:
You could do this 

```suggestion
        for (Source source : style.getSources()) {
```

Ideally you use the 

```java
mapboxMap.getStyle(new Style.OnStyleLoaded() {
      @Override
      public void onStyleLoaded(@NonNull Style style) {

     ATTRIBUTION CODE HERE

      }
    });
```

a couple of lines above so that you don't need the `if (style != null) {`